tests/foreman/ui/test_reporttemplates.py

```diff
@@ -562,3 +566,54 @@ def test_positive_gen_entitlements_reports_multiple_formats(
             tree_result = etree.tostring(tree.getroot(), pretty_print=True, method='html').decode()
         assert client.hostname in tree_result
         assert DEFAULT_SUBSCRIPTION_NAME in tree_result
+
+
+@pytest.mark.tier3
+def test_positive_generate_all_installed_packages_report(
+    session, setup_content, rhel7_contenthost, target_sat
+):
+    """Generate an report using the 'Host - All Installed Packages' Report template
+
+    :id: 63ab3246-0fc5-48b3-ba56-7becfa0c5a7b
+
+    :setup: Installed Satellite with Organization, Activation key,
+            Content View, Content Host, and custom product with installed packages
+
+    :steps:
+        1. Monitor -> Report Templates
+        2. Host - All Installed Packages -> Generate
+        3. Select Date, Output format, and Hosts filter
+
+    :expectedresults: A report is generated containing all installed package
+            information for a host
+
+    :BZ: 1826648
+
+    :customerscenario: true
+    """
+    org, ak, cv, lce = setup_content
+    setup_org_for_a_custom_repo(
+        {
+            'url': settings.repos.yum_6.url,
+            'organization-id': org.id,
+            'content-view-id': cv.id,
+            'lifecycle-environment-id': lce.id,
+            'activationkey-id': ak.id,
+        }
+    )
+    client = rhel7_contenthost
+    client.install_katello_ca(target_sat)
+    client.register_contenthost(org.label, ak.name)
+    assert client.subscribed
+    client.execute(f'yum -y install {FAKE_0_CUSTOM_PACKAGE_NAME} {FAKE_1_CUSTOM_PACKAGE}')
+    with session:
+        session.location.select('Default Location')
```

Comment:
You didnt select the org, is it redirecting the right org view or its not necessary ?

Reply:
the Correct org is already selected when the session is launched